classes/plain.lua

```diff
@@ -34,7 +34,14 @@ plain.defaultFrameset = {
 plain.firstContentFrame = "content"
 
 plain.options.direction = function (value)
-  if value then plain.defaultFrameset.content.direction = value end
+  SILE.documentState.direction = value
+  if value then
+    for id, frame in pairs(plain.defaultFrameset) do
```

Comment:
Shouldn't rather the frame initialization logic look for the preferred direction (in a setting ?), rather than forcing it from here? I am concerned it may likely break for all dynamically generated frames created _after_ class initialization (e.g. new frames created programmatically by commands - I used them in my "parbox" implementation, but as of the core SILE, there's also the "frametricks" package that can be used to do such things.)